src/BizHawk.Common/IsExternalInit.cs

```diff
@@ -0,0 +1,4 @@
+namespace System.Runtime.CompilerServices
+{
+	public static class IsExternalInit {}

```

Comment:
Is this kosher? The `init` auto-props you added aren't really an improvement over using named arguments w/ a ctor anyway.